js/coinex.js

```diff
@@ -1285,6 +1286,72 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async fetchFundingHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchFundingHistory() requires a symbol argument');
+        }
+        limit = (limit === undefined) ? 100 : limit;
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+            'limit': limit,
```

Comment:
@Dan-krm have you tried this endpoint without a `limit` param? If not, please check that, cause we don't send the `offset` here, so I suspect the `limit` is also optional.

Reply:
I get [ExchangeError] Invalid Parameter if I leave 'limit' undefined

Reply:
@Dan-krm ok, got it. Anyways, it's worth checking if a param is really required in cases like this (just a sidenote).

Reply:
> @Dan-krm ok, got it. Anyways, it's worth checking if a param is really required in cases like this (just a sidenote).

Okay I'll keep that in mind, I think Coinex might have a few cases like this